test/AutoRest.TestServer.Tests/Tests/body-complex.cs

```diff
@@ -6,7 +6,7 @@
 using body_complex.Models.V20160229;
 using NUnit.Framework;
 
-namespace AutoRest.TestServer.Tests
+namespace AutoRest.TestServer.Tests.Tests
```

Comment:
This is a strange namespace.

Reply:
Yup. The tests are in a Tests namespace. But, the project also has 'Tests' in the name of it. It made sense to put the tests into their own namespace since we'll end up with many of them. Then, the files at root are test infrastructure only. I could rename the namespace to 'UnitTests' if that is less weird.

Reply:
Can we just not moving tests into `Tests` subdirectory? It's very clear they are tests.

Reply:
I would rather move infrastructure into subfolder,

Reply:
Done.